.ci/resources.yaml

```
@@ -0,0 +1,63 @@
+---
+name: 'admission-controller'
+version: '${COMPONENT_VERSION}'
+type: 'ociImage'
+relation: 'local'
+access:
+  type: 'ociRegistry'
+  imageReference: 'eu.gcr.io/gardener-project/gardener/admission-controller:${COMPONENT_VERSION}'
+---
+---
```

Why we need this separator 2 times?

```suggestion
---
```